test/Garnet.test/RespAdminCommandsTests.cs

```diff
@@ -217,6 +217,56 @@ public void SeSaveRecoverObjectTest()
                 Assert.AreEqual(ldata, returnedData);
             }
         }
+        [Test]
+        [TestCase(63, 15, 1)]
+        [TestCase(63, 1, 1)]
+        [TestCase(16, 16, 1)]
+        [TestCase(5, 64, 1)]
+        public void SeSaveRecoverMultipleObjectsTest(int memorySize, int recoveryMemorySize, int pageSize)
+        {
+            string sizeToString(int size) => size + "k";
+            bool useAzure = false;
```

Comment:
see "[Values] bool useAzure" in other tests. Also, you don't need to test it call IgnoreIfNotRunningAzureTests here; TestUtils.CreateGarnetServer does that.